src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
+@import "./variables";
+
+* {
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 198px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__content-wrapper {
+    padding: 32px 16px 16px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__img {
+    width: 100%;
+    max-width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__title {
+    color: $mainColor;
+    font-family: $fontFamily;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+    padding-top: 40px;
+  }
+
+  &__code {
+    color: $secondaryColor;
+    font-family: $fontFamily;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0;
+    padding-top: 4px;
+  }
+
+  &__review-container {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 16px;
+  }
+
+  &__review {
+    color: $mainColor;
+    text-align: right;
+    font-family: $fontFamily;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 24px;
+    padding-bottom: 16px;
+    // p-b because margin-top wouldnt do the job 🙄 on button-link element
+
+    &-name {
+      color: $secondaryColor;
+      font-family: $fontFamily;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-amount {
+      color: $mainColor;
+      text-align: right;
+      font-family: $fontFamily;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button-link {
+    color: $buttonWhiteColor;
+    text-align: center;
+    font-family: $fontFamily;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition: 0.3s;
+    background-color: $buttonBlueColor;
+    padding: 11px 69px;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    &:hover {
+      color: $buttonBlueColor;
+      border: 1px solid $buttonBlueColor;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
9. [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover